Lack of Groovy(build.gradle) explanation for Compose Preview Support

https://github.com/takahirom/roborazzi/tree/0b0143ea4d4f1c3c2ab307db27e57d0a505f040a?tab=readme-ov-file#experimental-compose-preview-support

The README.md for Experimental Compose Preview Support does not include instructions on how to enable the Compose Preview Support using Groovy.

Applying the same syntax as kts in Groovy results in an error, so I believe it is necessary to add an example for Groovy as well.

```groovy
// In build.gradle
roborazzi {
    generateComposePreviewRobolectricTests {
        enable = true
        packages = ....
        .....
    }
}
```

Error message
```
Caused by: groovy.lang.MissingPropertyException: Could not set unknown property 'enable' for extension 'roborazzi' of type io.github.takahirom.roborazzi.RoborazziExtension.
```


https://github.com/takahirom/roborazzi/pull/488
I have added this in the pull request mentioned above. Please review it.

--- a/README.md
+++ b/README.md
@@ -991,10 +991,11 @@ After that, you can run the `recordRoborazziDebug` task to generate screenshots 
 
 ### Customizing the Preview screenshot test
 
-You can customize the generated test by adding the following configuration to your `build.gradle` file:
+You can customize the generated test by adding the following configuration to your `build.gradle.kts` file:
 
 ```kotlin
 roborazzi {
+  @OptIn(ExperimentalRoborazziApi::class)
   generateComposePreviewRobolectricTests {
     enable = true
     // The package names to scan for Composable Previews.
@@ -1012,6 +1013,30 @@ roborazzi {
   }
 }
 ```
+
+<details>
+<summary>build.gradle version</summary>
+<br>
+
+```groovy
+roborazzi {
+    generateComposePreviewRobolectricTests.enable.set(true)
+    // The package names to scan for Composable Previews.
+    generateComposePreviewRobolectricTests.packages.set(["com.example"])
+    // robolectricConfig will be passed to Robolectric's @Config annotation in the generated test class.
+    // See https://robolectric.org/configuring/ for more information.
+    generateComposePreviewRobolectricTests.robolectricConfig.set([
+        "sdk": "[32]",
+        "qualifiers": "RobolectricDeviceQualifiers.Pixel5",
+    ])
+    // If true, the private previews will be included in the test.
+    generateComposePreviewRobolectricTests.includePrivatePreviews.set(true)
+    // The fully qualified class name of the custom test class that implements [com.github.takahirom.roborazzi.ComposePreviewTester].
+    generateComposePreviewRobolectricTests.testerQualifiedClassName.set("com.example.MyCustomComposePreviewTester")
+}
+```
+
+</details>
 
 ## Manually adding Compose Preview screenshot tests
 

--- a/README.md
+++ b/README.md
@@ -1014,29 +1014,11 @@ roborazzi {
 }
 ```
 
-<details>
-<summary>build.gradle version</summary>
-<br>
-
-```groovy
-roborazzi {
-    generateComposePreviewRobolectricTests.enable.set(true)
-    // The package names to scan for Composable Previews.
-    generateComposePreviewRobolectricTests.packages.set(["com.example"])
-    // robolectricConfig will be passed to Robolectric's @Config annotation in the generated test class.
-    // See https://robolectric.org/configuring/ for more information.
-    generateComposePreviewRobolectricTests.robolectricConfig.set([
-        "sdk": "[32]",
-        "qualifiers": "RobolectricDeviceQualifiers.Pixel5",
-    ])
-    // If true, the private previews will be included in the test.
-    generateComposePreviewRobolectricTests.includePrivatePreviews.set(true)
-    // The fully qualified class name of the custom test class that implements [com.github.takahirom.roborazzi.ComposePreviewTester].
-    generateComposePreviewRobolectricTests.testerQualifiedClassName.set("com.example.MyCustomComposePreviewTester")
-}
-```
-
-</details>
+> **Note**  
+> If you are using build.gradle instead of build.gradle.kts, you need to use the set method for each assignment, like
+> ```kotlin
+> generateComposePreviewRobolectricTests.packages.set(["com.example"])
+> ```
 
 ## Manually adding Compose Preview screenshot tests
 

--- a/docs/topics/preview_support.md
+++ b/docs/topics/preview_support.md
@@ -22,10 +22,11 @@ After that, you can run the `recordRoborazziDebug` task to generate screenshots 
 
 ### Customizing the Preview screenshot test
 
-You can customize the generated test by adding the following configuration to your `build.gradle` file:
+You can customize the generated test by adding the following configuration to your `build.gradle.kts` file:
 
 ```kotlin
 roborazzi {
+  @OptIn(ExperimentalRoborazziApi::class)
   generateComposePreviewRobolectricTests {
     enable = true
     // The package names to scan for Composable Previews.
@@ -43,6 +44,31 @@ roborazzi {
   }
 }
 ```
+
+<details>
+<summary>build.gradle version</summary>
+<br>
+
+```groovy
+roborazzi {
+    generateComposePreviewRobolectricTests.enable.set(true)
+    // The package names to scan for Composable Previews.
+    generateComposePreviewRobolectricTests.packages.set(["com.example"])
+    // robolectricConfig will be passed to Robolectric's @Config annotation in the generated test class.
+    // See https://robolectric.org/configuring/ for more information.
+    generateComposePreviewRobolectricTests.robolectricConfig.set([
+        "sdk": "[32]",
+        "qualifiers": "RobolectricDeviceQualifiers.Pixel5",
+    ])
+    // If true, the private previews will be included in the test.
+    generateComposePreviewRobolectricTests.includePrivatePreviews.set(true)
+    // The fully qualified class name of the custom test class that implements [com.github.takahirom.roborazzi.ComposePreviewTester].
+    generateComposePreviewRobolectricTests.testerQualifiedClassName.set("com.example.MyCustomComposePreviewTester")
+}
+```
+
+</details>
+
 
 ## Manually adding Compose Preview screenshot tests
 

--- a/docs/topics/preview_support.md
+++ b/docs/topics/preview_support.md
@@ -45,30 +45,11 @@ roborazzi {
 }
 ```
 
-<details>
-<summary>build.gradle version</summary>
-<br>
-
-```groovy
-roborazzi {
-    generateComposePreviewRobolectricTests.enable.set(true)
-    // The package names to scan for Composable Previews.
-    generateComposePreviewRobolectricTests.packages.set(["com.example"])
-    // robolectricConfig will be passed to Robolectric's @Config annotation in the generated test class.
-    // See https://robolectric.org/configuring/ for more information.
-    generateComposePreviewRobolectricTests.robolectricConfig.set([
-        "sdk": "[32]",
-        "qualifiers": "RobolectricDeviceQualifiers.Pixel5",
-    ])
-    // If true, the private previews will be included in the test.
-    generateComposePreviewRobolectricTests.includePrivatePreviews.set(true)
-    // The fully qualified class name of the custom test class that implements [com.github.takahirom.roborazzi.ComposePreviewTester].
-    generateComposePreviewRobolectricTests.testerQualifiedClassName.set("com.example.MyCustomComposePreviewTester")
-}
-```
-
-</details>
-
+> **Note**  
+> If you are using build.gradle instead of build.gradle.kts, you need to use the set method for each assignment, like
+> ```kotlin
+> generateComposePreviewRobolectricTests.packages.set(["com.example"])
+> ```
 
 ## Manually adding Compose Preview screenshot tests
 

